core/templates/dev/head/components/fallbackEditor.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directives for the fallback editor.
+ *
+ * @author sean@seanlip.org (Sean Lip)
+ */
+
+oppia.directive('fallbackEditor', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      trigger: '=',
+      getOnSaveFn: '&onSave',
+      outcome: '='
+    },
+    templateUrl: 'components/fallbackEditor',
+    controller: [
+      '$scope', 'editabilityService', function($scope, editabilityService) {
+        $scope.isEditable = editabilityService.isEditable();
+
+        $scope.editFallbackForm = {};
+        $scope.triggerEditorIsOpen = false;
+
+        $scope.INT_FORM_SCHEMA = {
+          type: 'int',
+          ui_config: {},
+          validators: [{
+            id: 'is_at_least',
+            min_value: 1
+          }]
+        };
+
+        $scope.triggerMemento = null;
```

Comment:
Mm, not easily. We do have a memento for the whole fallbacks object, but that's not granular enough. I think what I've done here is the simplest for now -- but @BenHenning, we should probably revisit how these things are handled when we do the frontend refactor.